test/system/src/test/java/io/pravega/test/system/DynamicRestApiTest.java

```diff
@@ -114,11 +107,7 @@ public void listScopes() {
         Client client = ClientBuilder.newClient();
         String responseAsString = null;
 
-        ClientConfig clientConfig = ClientConfig.builder()
-                .controllerURI(controllerGRPCUri)
-                .credentials(new DefaultCredentials(Utils.PRAVEGA_PROPERTIES.get("pravega.client.auth.username"),
```

Comment:
The constructor follows the pattern ` public DefaultCredentials(@NonNull String password, @NonNull String userName)` 
and the current code 
- does not pass the password but username twice
- and the property is not set for a system test run without auth.